modules/fedimint-ln-client/src/pay.rs

```diff
@@ -258,6 +287,40 @@ impl LightningPayFunded {
     async fn gateway_pay_invoice(
         gateway: LightningGateway,
         payload: PayInvoicePayload,
+    ) -> Result<String, GatewayPayError> {
+        // If we don't succeed to reach the gateway in 30 seconds we'd rather abort the
+        // payment so the user isn't surprised by a heavily delayed payment
```

Comment:
ChatGPT suggests to rewrite this as:
```suggestion
        // Abort the payment if we can't reach the gateway within 30 seconds
        // to prevent unexpected delays for the user.

```